File: src/main/java/org/scuola247/desktop/entity/Role.java
package org.scuola247.desktop.entity;

import ch.rasc.extclassgenerator.Model;

@Model(value = "Desktop.model.Role", readMethod = "userService.readRoles")
public class Role {

	private String name;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

}

File: src/main/webapp/app/model/Role.js
Ext.define("Desktop.model.Role",
{
  extend : "Ext.data.Model",
  fields : [ {
    name : "name",
    type : "string"
  } ],
  proxy : {
    type : "direct",
    directFn : "userService.readRoles"
  }
});